modules/sdk-coin-avaxp/src/lib/transactionBuilder.ts

```diff
@@ -121,129 +110,20 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
     return this;
   }
 
-  /**
-   * Getter for know if build should sign
-   */
-  get hasSigner(): boolean {
-    return this._signer !== undefined && this._signer.length > 0;
-  }
+  // Implement:
+  //  buildImplementation
+  //  signImplementation
+  //  get transaction
+  //  set transaction
+  //  validateRawTransaction
 
   /** @inheritdoc */
-  protected get transaction(): Transaction {
+  protected fromImplementation(rawTransaction: string): Transaction {
+    const [tx] = pvmSerial.AddPermissionlessValidatorTx.fromBytes(
+      Buffer.from(rawTransaction, 'hex'),
+      avmSerial.getAVMManager().getDefaultCodec()
+    );
+    this.initBuilder(tx);
```

Comment:
I think it works because initBuilder expects DeprecatedTx | pvmSerial.BaseTx | UnsignedTx;
and AddPermissionlessValidatorTx extends AvaxTx which has BaseTx.